active_learning_single_annotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,639 @@
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch 0:  87%|████████████████████████████████████████████████████████████████████▏         | 271/310 [00:48<00:07,  5.55it/s, loss=5.49, v_num=]\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:automm:Start to fuse 2 checkpoints via the greedy soup algorithm.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Predicting DataLoader 0: 100%|███████████████████████████████████████████████████████████████████████████████████| 13/13 [00:02<00:00,  5.89it/s]\n",
+      "Predicting DataLoader 0: 100%|███████████████████████████████████████████████████████████████████████████████████| 13/13 [00:02<00:00,  5.80it/s]\n",
+      "Predicting DataLoader 0: 100%|███████████████████████████████████████████████████████████████████████████████████| 13/13 [00:02<00:00,  5.71it/s]\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:automm:Models and intermediate outputs are saved to /home/ubuntu/examples/active_learning_single_annotator/AutogluonModels/ag-20230317_201329 \n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Predicting DataLoader 0: 100%|████████████████████████████████████████████████████████████████████████████████▋| 595/597 [01:40<00:00,  5.90it/s]"
+     ]
+    }
+   ],
+   "source": [
+    "predictor, pred_probs_unlabeled = predict_autogluon_classification( df_labeled,\n",
+    "                                                                    out_folder=None,\n",
+    "                                                                    df_predict=df_unlabeled,\n",
+    "                                                                    time_limit=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Obtain active learning scores\n",
+    "\n",
+    "Next, we will get the active learning scores for each datapoint. Since we only have one annotation per example, we only care about the active learning scores for the unlabeled data, the `active_learning_scores_unlabeled`. \n",
+    "\n",
+    "These scores represent how confident we are about an example's true label based on the currently obtained annotations; examples with the lowest scores are those for which additional labels should be collected (i.e. likely the most informative). These scores are estimated via **ActiveLab** algorithm developed by the Cleanlab team. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# hack active learning multiannotator with dummy pred_probs\n",
```

Comment:
Let's write this in it's final polished form, eg. just say:

"We don't need to provide `pred_probs` for our already-labeled examples, since we are not considering data re-labeling in this notebook (we collect at most one label per example here)."